app/models/ownership.rb

```diff
@@ -0,0 +1,6 @@
+class Ownership < ActiveRecord::Base
+	belongs_to :project
```

Comment:
again let's go for consistent 2-space indenting ... hmm not sure what that codeclimate issue is about - it's missing other things that I think are important.

Maybe the presence of the multiple has_many is messing with things?